Documentation/debugging/unix-instructions.md

```diff
@@ -6,10 +6,40 @@ CoreFX can be debugged on unix using both lldb and visual studio code
 ## Using lldb and SOS
 
 - Run the test using msbuild at least once with `/t:BuildAndTest`.
-- Install version 3.6 of lldb and launch lldb with corerun as the process and arguments matching the arguments used when running the test through msbuild.
+- Install version 3.6 of lldb and launch lldb with dotnet as the process and arguments matching the arguments used when running the test through msbuild.
 - Load the sos plugin using `plugin load libsosplugin.so`.
 - Type `soshelp` to get help. You can now use all sos commands like `bpmd`.
 
+## Debugging core dumps with lldb
+
+It is also possible to debug .NET Core crash dumps using lldb and SOS. In order to do this, you need all of the following:
+
+- A machine whose environment matches the one used to produce the crash dumps. For crash dumps occurring on CI machines, you can either
+  - Log onto a matching machine in the pool.
+  - Create a new VM from the matching image.
```

Comment:
I think it is impossible right now, but hopefully in the future the process around our CI machines and the images for them becomes more transparent and usable for community members.